### PR TITLE
Remove padding in between paragraph blocks for collection description

### PR DESCRIPTION
**Pull request by Robinnnnn**
**Reported Bug**
image

**Before**
- no padding in between paragraphs
- weird expanding behavior on-click

https://user-images.githubusercontent.com/12162433/199828898-02f0a0ba-a8de-4326-b38c-46c2d2d9d4d5.mov

**After**
- ensure padding
- no padding change on click
image

### CODE REVIEW

File: src/scenes/UserGalleryPage/UserGalleryCollection.tsx [hunk 1]
```diff
@@ -262,10 +262,6 @@ const StyledCollectorsNote = styled(BaseM)<{ showMore: boolean }>`
           }
 
           -webkit-line-clamp: 2;
-
```
Comment (tbezman) [inline src/scenes/UserGalleryPage/UserGalleryCollection.tsx:265]: @Robinnnnn we only want to remove this if we're tablet size and greater. logic is a bit hard to follow but basically the idea is if we're doing line clamping, we don't want the padding, and on > tablet, we don't do the line clamp

Comment (Robinnnnn) [reply src/scenes/UserGalleryPage/UserGalleryCollection.tsx:265]: thanks @tbezman, fixed! tested on both desktop + mobile
